ambassador/schemas/v0/Mapping.schema

```diff
@@ -63,6 +63,7 @@
         "path_redirect": { "type": "string" },
         "priority": { "type": "string" },
         "precedence": { "type": "integer" },
+        "remove_response_headers": { "type": "array", "items": { "type": "string" } },
```

Comment:
The v0 schema should really be considered frozen at this point. Let's drop it from here -- it's fine if this is a v1-only feature.